br/pkg/metautil/metafile.go

```diff
@@ -715,7 +720,7 @@ func (writer *MetaWriter) flushMetasV2(ctx context.Context, op AppendOp) error {
 	}
 
 	name := op.name()
-	writer.metafileSizes[name] += writer.metafiles.size
+	writer.metafileSizes[name] += writer.metafiles.sstFilesSize
```

Comment:
I think you can add a `dataFileSize` to `writer`, `writer.metafileSizes` should store the size of meta files.

Reply:
Done.